pkg/tests/observability_reconcile_test.go

```diff
@@ -53,6 +53,16 @@ var _ = Describe("Observability:", func() {
 		Expect(err).ToNot(HaveOccurred())
 
 		By("Waiting for MCO retentionResolutionRaw filed to take effect")
+		mcoCR, err := dynClient.Resource(utils.NewMCOGVRV1BETA2()).Get(MCO_CR_NAME, metav1.GetOptions{})
+		Expect(err).NotTo(HaveOccurred())
+		if _, adv := mcoCR.Object["spec"].(map[string]interface{})["advanced"]; !adv {
+			Skip("Skip the case since the MCO CR did not have advanced spec configed")
+		} else {
+			if _, rec := mcoCR.Object["spec"].(map[string]interface{})["advanced"].(map[string]interface{})["retentionConfig"]; !rec {
+				Skip("Skip the case since the MCO CR did not have advanced retentionConfig configed")
+			}
+		}
+
```

Comment:
suggest to have an util method to judge if the field in CR exists or not.

Reply:
updated